src/python/ddapp/ikplanner.py

```diff
@@ -1352,6 +1352,15 @@ def computePostureCost(self, pose):
         return np.sum(np.abs(pose - nominalPose)*cost)
 
 
+    def runMultiRRT(self, grHand, qStart, xGoal, objectGrasped = False):
```

Comment:
internally, it seems this function isn't genreally usable, so it probably shouldn't be here.  Perhaps it should be part of a specific task planner class like the tabledemo.py